docs/pages/develop/user-interface/splash-screen-and-app-icon.mdx

```diff
@@ -4,153 +4,123 @@ description: Learn how to add a splash screen and app icon to your Expo project.
+After creating a splash screen icon, export it as a **.png** and save it in the **assets/images** directory. By default, Expo uses **splash-icon.png** as the file name. If you decide to change the name of your splash screen file, make sure to use that in the next step.
 
-> **Note:** **Currently, only .png images are supported** to use as a splash screen in an Expo project. If you use another image format, making a production build of your app will fail.
+> **Note:** **Currently, only .png images are supported** to use as a splash screen icon in an Expo project. If you use another image format, making a production build of your app will fail.
 
 </Step>
 
 <Step label="3">
 
-### Add the splash screen in app config
+### Configure the splash screen icon
 
-Open the app config and add the local path as the value of [`splash.image`](/versions/latest/config/app/#image) property to point to your new splash image.
+Open the app config file, under plugins set the corresponding properties as follows:
 
 ```json app.json
 {
-  "splash": {
-    /* @info */
-    "image": "./assets/images/splash.png"
-    /* @end */
+  "expo": {
+    "plugins": [
+      [
+        "expo-splash-screen",
+        {
+          /* @info The background color of the splash screen */
+          "backgroundColor": "#232323",
+          /* @end */
+          /* @info The path to the splash screen icon */
+          "image": "./assets/splash-icon.png",
+          /* @end */
+          /* @info The path to the splash screen icon for dark mode (optional) */
+          "dark": {
+            "image": "./assets/splash-icon-dark.png",
+            /* @end */
+            /* @info The background color of the splash screen for dark mode */
+            "backgroundColor": "#000000"
+            /* @end */
+          },
+          /* @info The width of the splash screen icon */
+          "imageWidth": 200
+          /* @end */
+        }
+      ]
+    ]
   }
 }
 ```
 
-You can test your new splash image by starting the development server with `npx expo start` and using a development build or Expo Go to preview it.
+To test your new splash screen, create a [preview](/build/eas-json/#preview-builds) or [production](/build/eas-json/#production-builds) build of your app.
 
-> **Note:** In development mode, when you test your new splash screen, you will notice an information bar at the bottom of the screen. It displays the information about how much of the JavaScript code is downloaded on the device when preparing your app. It doesn't appear in production apps.
+> **warning** Do not use Expo Go to test your splash screen as Expo Go renders your app icon while the splash screen is visible. Instead, use a [preview build](/build/eas-json/#preview-builds) or a [production build](/build/eas-json/#production-builds).
```

Comment:
This warning only mentions preview and production builds. Does that mean development builds are also not supported?

Reply:
Brent said:

> we plan to update the loading screen in expo go and dev client soon, need to do a mockup for it. the idea is to indicate clearly that this is a loading screen and not splash screen.

Maybe we can mention that. 🤔